app/views/landing-pages/parents/PageParents.vue

```diff
@@ -646,6 +647,14 @@ export default {
     if (this.type === 'thank-you') {
       this.onClassBooked()
     }
+
+    const pricingWaypoint = $('.pricing-grid-container').waypoint({
+      offset: '85%',
+      handler: (direction) => {
+        pricingWaypoint.disable()
+        window.me.trackActivity('viewed-parents-pricing')
+      }
+    })[0]
```

Comment:
Should we select the first index of the `$('.pricing-grid-container')[0]`? Otherwise we may attach waypoints to multiple containers, then only disable the first one.
Unsure if this is a real issue so ignore if not relevant.

Reply:
Another thought is that while we are in the `handler` function we have access to the waypoint instance with `this`. Thus we should be able to call `this.disable()` without the variable `pricingWaypoint`.

It's mentioned in the documentation under the `this?` section: http://imakewebthings.com/waypoints/guides/getting-started/